### PR TITLE
Readme: add specific binary name a user should use

### PR DESCRIPTION
**Pull request by 0xc0170**
@c1728p9  @ashok-rao Please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ All supported boards mentioned above are set up to use bootloader with a CI test
 To either use a different board or a different SD card slot, you must rebuild the bootloader and update the application.
 To do this:
 
-1. Create a bootloader for your board in the project [mbed-os-example-bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader).
-2. Update your board to use the newly created booloader image. To do this, set the target value `bootloader_img` to the file path of the bootloader image.
+1. Create a bootloader for your board in the project [mbed-os-example-bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader). The bootloader imagine is named ``mbed-os-example-bootloader.bin``.
```
Comment (ashok-rao) [inline README.md]: typo: imagine -> "image".

File: README.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ All supported boards mentioned above are set up to use bootloader with a CI test
...
+2. Update your board to use the newly created bootloader image. To do this, set the target value `bootloader_img` to the file path of the bootloader image.
 
 ```
     "target_overrides": {
```
Comment (ashok-rao) [inline README.md]: Are we planning to remove "NUCLEO_F429ZI" hard code and generalize it more to say something like <your_bootloader_here>.img or similar ??

**Review by ashok-rao — CHANGES_REQUESTED** [review]
Requesting minor changes. Also, @c1728p9 ..any plans to add more details to the troubleshooting section? (say) where to find the right "restrict_size", how to check VTOR for any changes due to flash algo etc?

File: README.md (hunk 2 above)
Comment (0xc0170) [reply README.md]: Can you pinpoint what lines should be changed and how? To propose changes here? the readme shows how to do it for one target (easy to replace the target name).

**Comment by ashok-rao** [comment]
We also need to add links to the FLASH IAP, flash porting guide & the bootloader docs..

**Comment by 0xc0170** [comment]
Rebased . That tutorial can be the entry point for bootloader (and we add links for flash IAP, etc) there - a meeting point. does that sound good?

https://github.com/ARMmbed/mbed-os-example-bootloader-blinky/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R5 https://github.com/ARMmbed/mbed-os-example-bootloader-blinky/pull/6/files#diff-04c6e90faac2675aa89e2176d2eec7d8R5

File: README.md (hunk 2 above)
Comment (ashok-rao) [reply README.md]: Current: "NUCLEO_F429ZI": {
                       "target.bootloader_img": "bootloader/NUCLEO_F429ZI.bin"

Proposed: In mbed_app.json, to add your newly built bootloader image to the application, make the following changes, replacing <target_name> with your board name and the new bootloader file name previously built:
"<target_name>": {
   "target.bootloader_img": "bootloader\\(Your_new_bootloader_filename).bin

Something like this?? Just to keep it more generic.. else in the future we will have a phase similar to the era of K64F .. but for the NUCLEO_F429ZI this time.. what do you think?

Comment (0xc0170) [reply README.md]: Fixed now

**Comment by ashok-rao** [comment]
Ok. just the troubleshooting section pending  now..

**Comment by 0xc0170** [comment]
>Ok. just the troubleshooting section pending now..

This section (=troubleshooting ) is part of the bootloader tutorial. That page plus related pages (flash IAP, etc) should cover it all (if not , lets fix it there).

**Comment by ashok-rao** [comment]
you mean to say https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/bootloader/ is where the troubleshooting section needs to go? .. If yes, then fine with me. Let's add more to the docs there rather than overpopulating the readme.

**Comment by 0xc0170** [comment]
> you mean to say https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/bootloader/ is where the troubleshooting section needs to go? .. If yes, then fine with me. Let's add more to the docs there rather than overpopulating the readme.

Yes, exactly, so any readme can point there to have it in one place. I'll add it tomorrow

**Comment by 0xc0170** [comment]
Requested changes - done - is that correct?

**Comment by ashok-rao** [comment]
I don't see the troubleshooting section on the bootloader docs page yet.. can you point me to a link if I'm looking in the wrong place?

**Comment by ashok-rao** [comment]
As discussed, these links and other sections will go into "bootloader tutorial" page.. This PR is good to go! :-)

**Review by ashok-rao — APPROVED** [review]

**Pull request merged**